play-screen.lua

```diff
@@ -56,14 +58,27 @@ function PlayScreen:update(dt)
 	table.foreach(platforms, function (i)
 		platforms[i]:update(dt)
 	end)
+
+	local numAlive = 0
+	local tempWinner = 0
 	table.foreach(players, function (i)
 		players[i]:update(dt)
 
-		if players[i].timer:getTime() <= 0 and players[i].controller:startButton() then
+		if players[i].alive then
```

Comment:
i think this should be declared at the start and decreased when someone dies, instead of recalculated every frame